deps/harfbuzz-icu-freetype

```diff
@@ -1 +1 @@
-Subproject commit 364b30bb26f3c4821e675f6b3027b4af74340ae4
+Subproject commit 1fe3240ba2b89a8cab9d7c4c586ecfbf66f18f03
```

Comment:
Also, the active target of this PR should be https://github.com/hjanetzek/alfons/tree/wip (the branch we currently use in tangram-es).